CHANGES

```diff
@@ -1,6 +1,9 @@
 Revision history for Shaderc
 
-v2019.12 2020-01-22
+v2020.0 2020-01-22
```

Comment:
Need "v2020.0-dev"
The -dev part is what makes it a development branch

Reply:
Fixed